cli_classic.c

```diff
@@ -555,6 +561,126 @@ int main(int argc, char *argv[])
 		goto out_shutdown;
 	}
 
+	if (lock){
+
+		unsigned char cmd[1] = {0x05};
+		unsigned char sr1, sr2;
+
+		msg_gdbg("1. Read SR1 and SR2\n");
+
+		//Read status register 1
+		cmd[0] = 0x05;
+		spi_send_command(fill_flash, sizeof(cmd), sizeof(sr1), cmd, &sr1);
+		msg_gdbg("SR1 = 0x%02x\n", sr1);
+
+		programmer_delay(100);
+
+		//Read status register 2
+		cmd[0] = 0x35;
+		spi_send_command(fill_flash, sizeof(cmd), sizeof(sr2), cmd, &sr2);
+		msg_gdbg("SR2 = 0x%02x\n", sr2);
+
+		if ((sr1 & 0x80) && (sr2 & 0x01)){
+			msg_ginfo("SPI flash is in One Time Program mode. Can't write to it anymore!\n");
```

Comment:
@piotr-kleins its useless to proceed when OTP mode is active already. Place `goto out_shutdown;
` under the msg_ginfo